app/src/main/java/edu/uc/mille5bm/smartnutrition/MainActivity.kt

```diff
@@ -10,7 +10,7 @@ class MainActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
         findViewById<View>(R.id.btn_nutrition).setOnClickListener { startActivity(Intent(applicationContext, NutritionInfoActivity::class.java)) }
-        findViewById<View>(R.id.btn_calorie).setOnClickListener { startActivity(Intent(applicationContext, CalorieCounterActivity::class.java)) }
+        findViewById<View>(R.id.btn_caloriepage).setOnClickListener { startActivity(Intent(applicationContext, CalorieCounterActivity::class.java)) }
```

Comment:
Changed the button name in the layout design.
Source:
https://www.youtube.com/watch?v=vXjJyLpydxg
https://www.youtube.com/watch?v=YaWmNhlOXYk

Reply:
Go a step further.  lowerCamelCase, without underscores, is convention.